src/components/Books/Book.css

```diff
@@ -10,30 +11,165 @@
   justify-content: space-between;
   width: 97%;
   background-color: #fff;
-  padding: 1.2rem;
+  padding: 1.2rem 6rem 1.2rem 1.2rem;
+  border: 1px solid #dcdcdc;
+}
+
+.functionalities {
+  width: 30%;
+}
+
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: Montserrat-bold, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin-bottom: 0.25rem;
+  padding: 0;
+}
+
+.book_title {
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: RobotoSlab-Bold, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  padding: 0;
+  width: 60%;
+}
+
+.author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--primary-color);
+}
+
+.functions {
+  margin-top: 1.4rem;
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.demacation {
+  background-color: #e8e8e8;
+  padding: 0.4px;
+}
+
+.big_demacation {
+  background-color: #e8e8e8;
+  padding: 1px;
+  height: 100px;
+}
+
+.progress-bar {
+  width: 65px;
+  height: 65px;
+  border-radius: 50%;
+
+  /* stylelint-disable-next-line csstree/validator */
+  background:
```

Comment:
- [ ] Kindly uncomment the linters and fix the errors the linter will highlight.